src/source.rs

```diff
@@ -164,7 +165,8 @@ fn indirect_sources(
                     .expect("strip prefix")
                     .to_owned()
             })
-            .filter(|rel_path| globset.as_ref().map_or(true, |gs| gs.is_match(rel_path)))
+            .filter(|rel_path| examine_globset.as_ref().map_or(true, |gs| gs.is_match(rel_path)))
+            .filter(|rel_path| exclude_globset.as_ref().map_or(true, |gs| !gs.is_match(rel_path)))
```

Comment:
We should document in the readme or option docstrings how these two interact. Matching inclusions before exclusions makes sense to me.